axon-framework/events/event-processors/streaming.md

```diff
@@ -587,6 +587,75 @@ public class AxonConfig {
 {% endtab %}
 {% endtabs %}
 
+#### Token Stealing
+
+As described at the [start](#tracking-tokens), streaming processor threads can "steal" tokens from one another.
+You may regard a token as stolen when a thread loses a [token claim](#token-claims).
+Situations like this internally result in an `UnableToClaimTokenException`, caught by both streaming event processor implementations and translated into warn- or info-level log statements.
+
+Where the framework uses token claims to ensure that a single thread is processing a sequence of events, it supports token stealing to ascertain event processing is not blocked forever.
```

Comment:
Maybe "guarantee" instead of "ascertain"?

Reply:
That's a good suggestion, thanks! Will adjust it.